keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -380,6 +595,59 @@ def inpaint(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def tokenize_prompt(self, prompt: str) -> tf.Tensor:
```

Comment:
Let's move this to the Tokenizer class, unless that's technically infeasible

Reply:
It seems pretty useful. What do you think @LukeWood? Should we open an issue?